src/community/l10n_community_roles.md

```diff
@@ -1,14 +1,24 @@
-# Roles within Mozilla l10n communities
+# Roles and community guidelines
 
-Within Mozilla l10n communities, each individual is organized into one of three roles: Locale manager, Translator, Contributor. These roles reflect the community structure within Pontoon as well. Each l10n community has its own unique implementation of these roles, but there is a baseline set of responsibilities and expectations that Mozilla places on those contributing to localization.
+Volunteer communities are the backbone of the localization (l10n) efforts at Mozilla. They are designed to be self-regulating and self-managing with minimal intervention from staff. For example, the responsibility of reviewing suggestions falls on the Translators and Managers within the community; they not only ensure quality and consistency but also keep an eye out for potential new volunteers to promote. And when Managers plan to step down, it’s crucial they develop succession plans to maintain continuity.
 
-This document defines the roles within each l10n community, as well as the responsibilities and expectations of each. We also define what the l10n-driver responsibilities and expectations are with regard to the community as a whole. Here we define responsibilities as what each individual commits to by accepting their role in the community. We define expectations as how you carry yourself in the role.
+The role of staff is to intervene only when necessary. The main reasons for staff to be involved is when self-regulation fails or there are violations of the [Community Participation Guidelines](https://www.google.com/url?q=https://www.mozilla.org/about/governance/policies/participation/&sa=D&source=editors&ust=1727922613565411&usg=AOvVaw3SJ7M4vfWka6Q-cV-l7QtW) (CPG). The goal is to empower the communities to thrive independently while maintaining the high standards that Mozilla is known for.
 
-## L10n-drivers (Mozilla staff)
+The localization team can occasionally take a more hands-on approach, proactively engaging with locales that are facing challenges. For example, it can happen when a member (or members) of a locale reaches out directly to localization staff to express concerns about their community. Other times, the team may receive product requests for specific locales, which prompts initiatives to ensure that these communities are capable of supporting the product expansion.
+
+In the following sections, we’ll review the current roles and expectations and suggest guidelines for when role criteria are not met — and how the localization staff can get involved and help in these different scenarios.
+
+## Roles: responsibilities and expectations
```

Comment:
I thought that we'd just keep the l10n-drivers section as is, since there wasn't any discussion about changing it or getting feedback on that section.
I do understand we want to stay away now from the term l10-drivers and I can update that - but did we want to get rid of that section entirely?

Reply:
Got rid of that section as per our 1:1 discussion